warden/app/app.go

```diff
@@ -82,6 +83,14 @@ const (
 var (
 	// DefaultNodeHome default home directories for the application daemon
 	DefaultNodeHome string
+
+	// ModuleBasics defines the module BasicManager is in charge of setting up basic,
+	// non-dependant module elements, such as codec registration
+	// and genesis verification.
+	ModuleBasics = module.NewBasicManager(
+		ibchooks.AppModuleBasic{},
```

Comment:
I think this should be added here instead:
https://github.com/warden-protocol/wardenprotocol/blob/f66d93d8518230b2719a9c0c0f46b118e4420f87/warden/app/app.go#L177

can you check if it works?

It should also make unnecessary to call `ModuleBasics.RegisterGRPCGatewayRoutes`